python/ray/_raylet.pyx

```diff
@@ -1666,6 +1667,10 @@ cdef class CoreWorker:
         return (CCoreWorkerProcess.GetCoreWorker().GetWorkerContext()
                 .CurrentActorIsAsync())
 
+    def current_serialized_runtime_env(self):
+        return (CCoreWorkerProcess.GetCoreWorker().GetWorkerContext()
+                .GetCurrentSerializedRuntimeEnv())
```

Comment:
I believe this will never change, right? Can we just store it as a python dict to avoid deserializing every time?

Reply:
Yep, makes sense